### ISSUE TITLE
Correct armv(7|8)l Jenkins node names

### ISSUE THREAD

**Issue by Willsparker** [AdoptOpenJDK/openjdk-infrastructure#1837]
Found as a part of https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/1716#issuecomment-755202080

As suggested by @sxa (https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/1806#pullrequestreview-562539987), we should change the Jenkins name of the following machines, so they match the inventory.yml, i.e. remove the `l` from `armv(7|8)l`:

- [ ]  build-packet-ubuntu1804-armv8l-1
- [ ]  docker-scaleway-ubuntu1604-armv7l-1

Should just be a case of changing their names, and restarting the Jenkins agent.

**Comment by Willsparker** [comment]
Clearing assignment as I don't have access to the machines to restart the Jenkins agent

**Comment by sxa** [comment]
I'm tempted to switch them all over to `aarch64` instead of `armv8` since I think that name is clearer to most people, and it's consistent with `uname` and what we use on the download site ... And the build job names

**Comment by Willsparker** [comment]
Yeah, that seems fair - should I rename the issue ? Should ARMv7 remain ARMv7? (I noticed that the job just refers to it as `arm`)

**Comment by Willsparker** [comment]
@sxa I'm happy to pick this up, as it's blocking me on #1716 , but presumably I'm going to need access to all of the `arm64` nodes, so I'm able to restart the Jenkins Agents.

(The following is a list of machines that I don't have access to. I do have it for the rest of the ARMv8 nodes in the inventory)

|Node in Inventory | IP | Other Info |
|--|--|--|
build-linaro-centos76-armv8-2 | 213.146.141.123 |
build-packet-ubuntu1804-armv8-1 | 139.178.82.234 | 
docker-packet-ubuntu1604-armv8-1 | 147.75.77.146 | 
docker-scaleway-ubuntu1604-armv7-1 | 51.158.73.136 | 
test-macstadium-macos11-arm64-1 | 199.7.163.51 | (Blocked by #1855 )  |

**Comment by Haroon-Khel** [comment]
@Willsparker Any update on this?

**Comment by Willsparker** [comment]
Still waiting on @sxa so I can get access to the above nodes

**Comment by sxa** [comment]
Quick update on this - the Linaro machines are not currently in production, the scaleway ones are going to get decomissioned (and are armv7 not armv8), the packet aarch64 ones listed above are also to be decomissioned soon. We'll need to consider what to do with the macos ones ...

**Comment by sxa** [comment]
Still considering keeping armv8 since aarch64 will become ambiguous across armv8, armv8.2 (Required by Dragonwell) and the upcoming armv9

**Comment by sxa** [comment]
After some discussions with someone at ARM I've come to the conclusion that standardising on `aarch64` is the right approach for the naming, so yes we should go for that :-)

**Comment by sxa** [comment]
The new ones added in https://github.com/adoptium/infrastructure/issues/2478 https://github.com/adoptium/infrastructure/issues/3586 have all been added as `armv8` or `armv8l` e.g.
- https://ci.adoptium.net/label/test-docker-ubi9-armv8l-1
- https://ci.adoptium.net/label/test-docker-ubi8-armv8-1
- https://ci.adoptium.net/label/test-docker-centosstream9-armv8-1

We should bump up the priority of this for usability/simplicity reasons - adding to Q4 plan.
